src/test/java/SignUpTest.java

```diff
@@ -15,6 +18,12 @@
 @SpringBootTest(classes = Application.class)
 public class SignUpTest {
 
+    String filename = "/Users/thomasbull/Downloads/Nevermore.png";
```

Comment:
No one will have the same file path thus this will break for others running this test